client/blocks/upgrade-nudge-expanded/index.jsx

```diff
@@ -137,7 +137,7 @@ UpgradeNudgeExpanded.propTypes = {
 
 const mapStateToProps = ( state, { plan = PLAN_PERSONAL } ) => ( {
 	plan: getPlanBySlug( state, plan ),
-	currentPlan: getCurrentPlan( state, getSelectedSiteId( state ) ),
+	currentPlan: getSitePlan( state, getSelectedSiteId( state ) ),
```

Comment:
Minor, but we can store the selectedSiteId in a variable since we use this twice.